barretenberg/cpp/src/barretenberg/polynomials/univariate.hpp

```diff
@@ -89,10 +89,13 @@ template <class Fr, size_t domain_end, size_t domain_start = 0, size_t skip_coun
     // Check if the univariate is identically zero
     bool is_zero() const
```

Comment:
This is kinda sketchy. Someone is definitely going to assume this means the univariate is zero, not just that it is zero outside of some range.

Reply:
@codygunton I'm not sure what you mean here. The intent was a genuine check on whether or no the univariate is identically zero - is this not doing that some for some reason?